src/modules/economy/components/MyExpenditures.tsx

```diff
@@ -68,24 +78,32 @@ export const MyExpenditures: React.FC<MyExpendituresProps> = ({
   }))
 
   return (
-    <Paper shadow={'lg'} p="md" style={{ overflowX: 'scroll' }}>
+    <>
       <Select
         label={'Periode'}
         data={dateRangeOptions}
         defaultValue={dateRangeOptions[0].value}
       />
+      <ResponsiveContainer width={'90%'} height={400}>
```

Comment:
The padding is kinda large on mobile

Reply:
tried making it smaller with every dumb way available, still no dice.

Reply:
Aight we can get back to this later